src/plugins/mf_priority.cpp

```diff
@@ -96,24 +101,29 @@ static void rec_update_cb (flux_t *h,
                            const flux_msg_t *msg,
                            void *arg)
 {
-    char *uid, *fshare, *bank, *default_bank;
+    char *uid, *fshare, *bank, *default_bank, *max_jobs;
 
-    if (flux_request_unpack (msg, NULL, "{s:s, s:s, s:s, s:s}",
+    if (flux_request_unpack (msg, NULL, "{s:s, s:s, s:s, s:s, s:s}",
                              "userid", &uid,
                              "bank", &bank,
                              "default_bank", &default_bank,
-                             "fairshare", &fshare) < 0) {
+                             "fairshare", &fshare,
+                             "max_jobs", &max_jobs) < 0) {
         flux_log_error (h, "failed to unpack custom_priority.trigger msg");
         goto error;
     }
 
     if (flux_respond (h, msg, NULL) < 0)
         flux_log_error (h, "flux_respond");
 
-    if (strcmp (bank, default_bank) == 0)
-        users[std::atoi (uid)]["default"] = std::stod (fshare);
+    struct bank_info b;
+
+    b.fairshare = std::atof (fshare);
+    b.max_jobs = std::atoi (max_jobs);
+    b.current_jobs = 0;
```

Comment:
Would it save a couple assignments to do something like:
```c
    struct bank_info *b;
    b = &users[std:atoi(uid)][bank];
    b->fairshare = std::atof (fshare);
    b->max_jobs = std::atoi (max_jobs);
    ...
```

Then you do not have to assign `b` to `users[std::atoi (uid)][bank]` below